src/config/enterprise.rs

```diff
@@ -290,13 +290,13 @@ impl TryFrom<&Config> for EnterpriseMetadata {
             DATADOG_REPORTING_PRODUCT
         );
 
-        // Get the configuration version. In DD Pipelines, this is referred to as the 'config hash'.
-        let config_version = value.version.clone().expect("Config should be versioned");
+        // Get the configuration version hash. In DD Pipelines, this is referred to as the 'config hash'.
+        let config_version_hash = value.version.clone().expect("Config should be versioned");
```

Comment:
Per https://datadoghq.atlassian.net/browse/OP-400, this can later be the expanded form too.